### PR TITLE
Add tw.ddnet.ddnet

### PR DESCRIPTION
**Pull request by ElXreno**

### CODE REVIEW

**Comment by ElXreno** [comment]
bot, build tw.ddnet.ddnet

**Comment by ElXreno** [comment]
bot, build tw.ddnet.ddnet

**Comment by ElXreno** [comment]
fsckObject disabled because repo is broken. Issue: https://github.com/ddnet/ddnet/issues/2196

**Comment by ElXreno** [comment]
bot, build tw.ddnet.ddnet

**Comment by heinrich5991** [comment]
Thanks for working on this! @ElXreno

**Comment by ElXreno** [comment]
bot, build tw.ddnet.ddnet

**Comment by barthalion** [comment]
bot, build tw.ddnet.ddnet

File: tw.ddnet.ddnet.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+id: tw.ddnet.ddnet
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+default-branch: stable
```
Comment (barthalion) [inline tw.ddnet.ddnet.yml:5]: Your manifest should define `command` as well.

Comment (ElXreno) [reply tw.ddnet.ddnet.yml:5]: Fixed. Thanks :+1:

**Pull request closed without merging**